modules/installation-initializing.adoc

```diff
@@ -330,6 +330,12 @@ ifdef::vsphere,vmc[]
 +
 The installation program connects to your vCenter instance.
 ... Select the data center in your vCenter instance to connect to.
++
```

Comment:
Hi @mjpytlak . I added this note because a user might notice some new fields in the updated install config file. I felt this note is a good place to send a user elsewhere if they need to configure a multiple vSphere datacenter environment. The note can be ignored and does not impede a typical user's progression in creating the installation file.

Reply:
After seeing this note in the published output, I see what you are doing here.